### PR TITLE
added UI changes

### PR DESCRIPTION
**Pull request by EugeneTeu**
Screenshot 2020-08-03 at 1 43 48 AM

Screenshot 2020-08-03 at 1 44 19 AM

### CODE REVIEW

File: frontend/src/logic/tasklogic.ts [hunk 1]
```diff
@@ -1,287 +1,317 @@
...
+        this.expectedDeliveryTime = expectedDeliveryTime
+        this.item = item
+        this.payerUid = payerUid
+        this.fee = fee
+        this.status = status
+        /*
     if (!Task.isValidState(uid, status)) {
       throw new Error("Missing doer for task with id " + id);
-    }
+    }*/
```
Comment (EugeneTeu) [inline frontend/src/logic/tasklogic.ts]: @pohlinwei This was causing issue when i was fetching on marketplace. can check ?

File: frontend/src/logic/requestlogic.ts [hunk 2]
```diff
@@ -1,296 +1,319 @@
...
+        throw new Error('Unable to get requests when uid is empty')
+    }
 
-  results.sort((prev, curr) => sortByReverseOrder(prev.task.expectedDeliveryTime, curr.task.expectedDeliveryTime))
-    .forEach(request => {
-      const status: Status = request.task.status;
+    const requestRef = db
+        .collection(COLLECTION_REQUESTS)
+        .where('uid', '==', uid)
+    //.where('status', 'in', [Status.OPEN, Status.PENDING])
```
Comment (EugeneTeu) [inline frontend/src/logic/requestlogic.ts]: @pohlinwei This status was a typo right? i commented it out

Comment (pohlinwei) [reply frontend/src/logic/requestlogic.ts]: Why do you think it is a typo? 😅

**Comment by pohlinwei** [comment]
By the way, the UI (as always) looks awesome! :D

**Review by pohlinwei — APPROVED** [review]
Yay! Almost done. :D Will be great if we can fix the code style. 😂

**Pull request merged**